README.md

```diff
@@ -1,75 +1,203 @@
+   ros2 control load_start_controller forward_command_controller_position
+   ```
+
+4. Run the test node:
+
+   Run a test node which will publish joint commands on /forward_command_controller_position/commands (std_msgs::msg::Float64MultiArray) after checking the current joint states (to create minimal increment for safety).
+   The node commands increment of 0.1 radians for each joint.
+   The commands are incremented in regards to /joint_states found when the node is run.
+
+   ```
+   ros2 run ur_ros2_control_demos test_driver
+   ```
+
+
+# How to use this Package and ROS Introduction
+
+ - [Install and Build](#install-and-build)
+   * [Install ROS Foxy and Development Tooling](#install-ros-foxy-and-development-tooling)
+   * [Setup ROS Workspace](#setup-ros-workspace)
+   * [Configure and Build Workspace](#configure-and-build-workspace)
+ - [Running Executables](#running-executables)
+   * [Using the Local Workspace](#using-the-local-workspace)
+ - [Testing and Linting](#testing-and-linting)
+ - [Creating a new ROS2 Package](#creating-a-new-ros2-package)
+ - [References](#references)
+
+## Install and Build
+
+### Install ROS Foxy and Development Tooling
+
+These instructions assume you are running Ubuntu 20.04:
+
+1. [Install ROS2 Foxy](https://index.ros.org/doc/ros2/Installation/Foxy/Linux-Install-Debians/).
+   You can stop following along with the tutorial after you complete the section titled: [Environment setup](https://index.ros.org/doc/ros2/Installation/Foxy/Linux-Install-Debians/#environment-setup).
+   Make sure you setup your environment with:
+   ```
+   source /opt/ros/foxy/setup.bash
+   ```
+
+   > **NOTE:** You may want to add that line to your `~/.bashrc`
+
+   > **NOTE:** There is also a `zsh` version of the setup script.
+
+1. [Install ROS2 Build Tools](https://index.ros.org/doc/ros2/Installation/Foxy/Linux-Development-Setup/#install-development-tools-and-ros-tools).
+   You do not need to build ROS2 from source.
+   Simply install the tooling under the section titled "Install development tools and ROS tools".
+
+1. Install `ccache`:
+   ```
+   sudo apt install ccache
+   ```
+
+1. Setup `colcon mixin` [Reference](https://github.com/colcon/colcon-mixin-repository) for convinience commands.
+   ```
+   sudo apt install python3-colcon-mixin
+   colcon mixin add default https://raw.githubusercontent.com/colcon/colcon-mixin-repository/master/index.yaml
+   colcon mixin update default
+   ```
+
+### Setup ROS Workspace
+
+1. Create a colcon workspace:
+   ```
+   export COLCON_WS=~/workspace/ros_ws_foxy
+   mkdir -p $COLCON_WS/src
+   ```
+
+   > **NOTE:** Feel free to change `~/workspace/ros_ws_foxy` to whatever absolute path you want.
+
+   > **NOTE:** Over time you will propably have multiple ROS workspaces, so it makes sence to them all in a subfolder.
+     Also, it is good practice to put the ROS version in the name of the workspace, for different tests you could just add a suffix to the base name `ros_ws_foxy`.
```

Comment:
I think this is too much information. Would prefer to delete it.

sence -> sense